src/utils/body.ts

```diff
@@ -18,20 +19,37 @@ export type ParseBodyOptions = {
    * parseBody should return { file: ['aaa', 'bbb'], message: 'hello' }
    */
   all?: boolean
```

Comment:
We can change these to be required and in the options, we can use `Partila<ParseBodyOptions>` to get them. This will provide a better type completion in the code.